ansible/roles/build-ossec-deb-pkg/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
   with_items: build_ossec_deb_pkg_dependencies
 
 - name: extract OSSEC archive to /tmp
-  shell: tar -xvf {{ build_path }}/{{ download_name }} -C /tmp
+  shell: tar -xvf {{ build_path }}/ossec-hids-{{ download_name }} -C /tmp
```

Comment:
Do you also need to update the other locations where `{{ download_name }}` is used in the task? Why did you move `ossec-hids` from `download_name` to inside the shell call?